### DISCUSSION TITLE
is it possible to set right-click copy when select something?

### DISCUSSION THREAD

**Discussion by NEX-S** [kovidgoyal/kitty#8177] [Q&A]
i want to set right-click copy when i selected something and right-click paste when nothing is selected

current copy_on_select might trigger by accident :(

exp:
select something -> right-click -> copy and exit select mode -> right-click -> paste

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#mouse-actions

**Reply by NEX-S** [reply]
sorry, i check the doc and try this
```
mouse_map right press grabbed copy_to_clipboard
mouse_map right press ungrabbed paste_from_clipboard
```
dont know why it didn't work :(

**Reply by zzhaolei** [reply]
this work:
```
mouse_map right        press grabbed,ungrabbed copy_to_clipboard
```

**Comment by kovidgoyal** [comment]
grabbed and ungrabbed refer to when the mouse is grabbed by the program
running in the terminal. And I dont know what you are trying to do.
Those mappings are sayin you want to copy to clipboard on button release
and paste on button press.
